locales/en-US/server.ftl

```diff
@@ -136,7 +136,10 @@ shotPageDMCAIncludeLink = Please include the URL of this shot in your email: { $
 shotPageKeepFor = How long should this shot be retained?
 // Note: shotPageSelectTime is a placeholder label for the time selection dropdown.
 shotPageSelectTime = Select time
-shotPageKeepIndefinitely = Indefinitely
+// The ∞ is used to indicate that the shot won't expire. It is also used in
+// shotIndexNoExpirationSymbol. Try to use the same symbol in both strings, or
+// if no such symbol is available for a language/culture, simply leave it out.
+shotPageKeepIndefinitely = Indefinitely ∞
```

Comment:
Ugh, I completely missed that you changed an existing string. Was it like this from the beginning? All locales won't pick up the change without a new string ID

Reply:
It was just `Indefinitely` before the patch.

Reply:
Yes, what I (likely) missed is that the string changed but the ID didn't
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings

Unfortunately, we need a patch that changes the ID, to force all locales to look at that string and retranslate it.